Extensions/PrimitiveDrawing/ShapePainterObject.cpp

```diff
@@ -72,6 +73,10 @@ void ShapePainterObjectBase::DoUnserializeFrom(
       element.GetChild("absoluteCoordinates", 0, "AbsoluteCoordinates")
           .GetValue()
           .GetBool();
+  clearAtEachFrame =
+      element.GetChild("clearAtEachFrame", 0, "ClearAtEachFrame")
+          .GetValue()
+          .GetBool();
```

Comment:
You can just do `element.GetChild("clearBetweenFrames").GetValue().GetBool()` (no need for the third parameter because there is no compatibility to ensure with older serialised value that would have another name).

This being said, we should take care *here* I think of the value to set if there is no "clearBetweenFrames". Because for projects that don't have it, we want it to be `true`, for backward compatibility.
So:

```
clearBetweenFrames = element.HasChild("clearAtEachFrame") ? element.GetChild("clearBetweenFrames").GetValue().GetBool() : true;
```